zebedee-reader/src/main/java/com/github/onsdigital/zebedee/search/fastText/FastTextHelper.java

```diff
@@ -0,0 +1,34 @@
+package com.github.onsdigital.zebedee.search.fastText;
+
+import java.nio.ByteBuffer;
+import java.nio.DoubleBuffer;
+import java.util.Base64;
+
+import static com.github.onsdigital.zebedee.util.VariableUtils.getVariableValue;
+
+public class FastTextHelper {
+
+    public static final String convertArrayToBase64(double[] array) {
+        final int capacity = 8 * array.length;
+        final ByteBuffer bb = ByteBuffer.allocate(capacity);
+        for (int i = 0; i < array.length; i++) {
+            bb.putDouble(array[i]);
+        }
+        bb.rewind();
+        final ByteBuffer encodedBB = Base64.getEncoder().encode(bb);
+        return new String(encodedBB.array());
+    }
+
+    public static double[] convertBase64ToArray(String base64Str) {
+        final byte[] decode = Base64.getDecoder().decode(base64Str.getBytes());
+        final DoubleBuffer doubleBuffer = ByteBuffer.wrap(decode).asDoubleBuffer();
+
+        final double[] dims = new double[doubleBuffer.capacity()];
+        doubleBuffer.get(dims);
+        return dims;
+    }
+
+    public static class Configuration {
```

Comment:
This should go in the configuration package with the other configuration

Reply:
Added `TODO - Fix me` as discussed